jlm/Makefile.sub

```diff
@@ -12,11 +11,6 @@ endef
 LLVM_TEST_GIT=$(LLVM_TEST_ROOT)/..
 LLVM_TEST_BUILD=$(LLVM_TEST_ROOT)/build
 
-
-.PHONY: install-lit
-install-lit:
-	sudo python3 /usr/lib/llvm-15/build/utils/lit/setup.py install
```

Comment:
Why remove it instead of updating the commands for installing lit?
        pip install "lit~=16.0"
        pip show lit

Reply:
Yes, I do not see why these commands are in the build system for the test suite. They should not be there.

Several other reasons why not just updating it:
1. `pip install` will also not work (so easily) on newer Ubuntu systems (as far as I remember). I rather rely on the user to install it and then maybe introduce a configure script to check whether it is there and set the correct paths for the build system.
2. We have other dependencies that are also not in the Makefile (and should not be there), such as jlm, clang, llvm

What we can talk about is maybe to document the dependencies (and later a configure script that checks whether they are there).